docs/user-guide/run-benchmark.md

```diff
@@ -280,9 +283,26 @@ When using HuggingFace datasets, you should always check if you need a `split`,
 }
 ```
 
-Then use: `--dataset-config config.json`
+**Benchmarking with large images:**
+When benchmarking with very large images, the pillow library throws an exception. To get around this, use a config with the argument "unsafe_allow_large_images", which disables the warning.
+
+```json
+{
+  "source": {
+    "type": "huggingface",
+    "path": "zhang0jhon/Aesthetic-4K",
+    "huggingface_kwargs": {
+      "split": "train"
+    }
+  },
+  "prompt_column": "text",
+  "image_column": "image",
+  "unsafe_allow_large_images": true
+}
+```
```

Comment:
Can we also add a usage of `prompt_lambda` in the config? For instance, [lmms-lab/LLaVA-OneVision-Data](https://huggingface.co/datasets/lmms-lab/LLaVA-OneVision-Data)

The configuration can't simply use `prompt_column`, but a `prompt_lambda: lambda x: x["conversations"][0]["value"] if len(x["conversations"]) > 1 else ""`

You can read how `prompt_lambda` is used and add a section for it.